src/static/js/fb-ctf.js

```diff
@@ -2464,6 +2501,41 @@ function setupInputListeners() {
       event.preventDefault();
       $(this).onlySiblingWithClass('active');
     });
+
+    // custom logo file selector
+    var $customEmblemInput = $('#custom-emblem-input');
+    var $customEmblemPreview = $('#custom-emblem-preview');
+    var $customEmblemCarouselNotice = $('#custom-emblem-carousel-notice');
+    $('#custom-emblem-link').on('click', function() {
+      $customEmblemInput.trigger('click');
+    });
+    // on file input change, set image preview and emblem carousel notice
+    $customEmblemInput.change(function() {
+console.log('foo');
+      var input = this;
+      if (input.files && input.files[0]) {
+        if (input.files[0].size > (500*1024)) {
```

Comment:
I think there may be some inconsistencies with this because in ```src/models/Logo.php``` the value of ```MAX_CUSTOM_LOGO_SIZE_BYTES``` is 500000